crates/forge-runner/src/running.rs

```diff
@@ -198,20 +198,17 @@ pub fn run_test_case(
     runner_config: &Arc<RunnerConfig>,
     runner_params: &Arc<RunnerParams>,
 ) -> Result<RunResultWithInfo> {
+    // we need to do this "trick" to be compatible with the older scarb versions
     ensure!(
-        case.available_gas.is_none(),
-        "\n    Attribute `available_gas` is not supported\n"
+        case.available_gas != Some(0),
+        "\n    Make sure you use the scarb version where `available_gas` attribute is supported\n"
     );
-    let available_gas = Some(usize::MAX);
 
     let func = runner.find_function(case.name.as_str()).unwrap();
-    let initial_gas = runner
-        .get_initial_available_gas(func, available_gas)
-        .unwrap();
```

Comment:
This one is sierra gas, right? Do we need to remove this? It prob changes nothing, but it looks like an automatic calculation of the gas required for the top lvl VM to execute the test (it doesn't care about gas used by blockifier anyways)

Reply:
Yes, this is a sierra gas. Maybe I will revert this change because we may need this one in the future